api/aws/dynamodb/v1/dynamodb.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package clutch.aws.dynamodb.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/aws/dynamodb/v1;ddbv1";
+
+import "google/api/annotations.proto";
+import "validate/validate.proto";
+
+import "api/v1/annotations.proto";
+
+service DDBAPI {
+  rpc GetTable(GetTableRequest) returns (GetTableResponse) {
+    option (google.api.http) = {
+      post : "/v1/aws/dynamodb/getTable"
+      body : "*"
+    };
+    option (clutch.api.v1.action).type = READ;
+  }
+
+  rpc UpdateTableCapacity(UpdateTableCapacityRequest) returns (UpdateTableCapacityResponse) {
+    option (google.api.http) = {
+      post : "/v1/aws/dynamodb/updateTableCapacity"
+      body : "*"
+    };
+    option (clutch.api.v1.action).type = UPDATE;
+  }
+
+  rpc UpdateGSICapacity(UpdateGSICapacityRequest) returns (UpdateGSICapacityResponse) {
+    option (google.api.http) = {
+      post : "/v1/aws/dynamodb/updateGSICapacity"
+      body : "*"
+    };
+    option (clutch.api.v1.action).type = UPDATE;
+  }
+}
+
+message Table {
+  option (clutch.api.v1.id).patterns = {
+    type_url : "clutch.aws.dynamodb.v1.Table",
+    pattern : "{region}/{table_name}"
+  };
+
+  string table_name = 1;
+  string region = 2;
+  repeated GlobalSecondaryIndex global_secondary_indexes = 3;
+  ProvisionedThroughput table_capacity = 4;
```

Comment:
nit: since were already in the context of table lets drop the `table_` prefix.
```suggestion
  string name = 1;
  string region = 2;
  repeated GlobalSecondaryIndex global_secondary_indexes = 3;
  ProvisionedThroughput capacity = 4;
```

Reply:
I wont leave other comments for each occurrence in this file, but the same applies for the rest.